tests/parsing/FilterHelperTest.py

```diff
@@ -51,8 +51,8 @@ def test_apply_filter(self):
         self.assertIsNotNone(apply_filter_result)
 
     def test_apply_filters(self):
-        apply_filters_result = apply_filters([('language', 'C', 'Python'),
-                                              ('can_fix', 'syntax')])
+        apply_filters_result = apply_filters({'language': ('C', 'Python'),
+                                              'can_fix': 'syntax'})
```

Comment:
How is this test passing when we need to have args for `can_fix` as a tuple rather than as a string?